content/en/docs/concepts/policy/_index.md

```diff
@@ -1,11 +1,69 @@
 ---
 title: "Policies"
 weight: 90
+no_list: true
 description: >
   Policies you can configure that apply to groups of resources.
```

Comment:
This now needs to change, as the section has a broader scope.

Reply:
updated.